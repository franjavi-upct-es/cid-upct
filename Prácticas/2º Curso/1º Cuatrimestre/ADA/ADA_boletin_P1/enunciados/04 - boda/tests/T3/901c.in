50
45 17
8 23 30 12 3 22 3 17 19
12 29 29 14 2 18 27 17 13 30 1 28 19
6 13 5 23 3 22 23
2 8 14
11 7 17 7 21 19 16 9 23 14 7 6
13 16 2 24 21 23 16 10 11 11 23 15 4 17
6 18 7 13 1 11 12
5 9 24 28 24 2
10 7 30 17 6 15 11 29 5 3 14
7 6 25 7 20 20 23 26
5 21 30 21 23 3
5 2 26 18 25 19
1 24
8 8 29 3 10 19 29 12 24
5 17 10 3 29 29
10 24 19 8 15 9 23 10 8 24 5
13 10 24 24 25 3 1 15 27 2 3 26 13 19
63 2
14 3 12 19 20 27 7 19 11 16 11 20 23 26 17
9 28 2 2 14 4 24 29 1 25
89 8
2 11 17
7 1 11 24 19 30 20 25
9 30 10 21 12 24 9 20 11 6
1 4
13 16 28 9 16 22 24 3 21 26 12 3 26 22
10 14 14 15 1 16 7 2 28 10 26
14 29 28 3 11 1 14 26 20 14 4 12 7 28 24
1 1
79 10
9 14 26 20 21 18 5 19 19 2
6 6 8 18 25 2 20
10 15 7 7 21 10 10 19 30 4 13
10 22 25 14 27 12 3 9 29 7 27
1 9
2 15 8
5 9 9 6 26 16
7 25 28 22 4 16 13 29
11 13 20 7 18 9 18 13 17 8 19 6
2 19 13
94 18
14 23 19 5 18 26 17 4 23 1 8 30 13 6 20
3 18 18 5
4 5 17 4 12
7 1 20 16 14 11 25 5
9 13 9 13 30 26 16 22 26 15
4 8 21 25 25
6 13 21 25 9 7 20
9 3 13 2 10 18 4 5 22 29
11 30 11 1 25 18 22 12 3 27 12 23
8 28 22 4 18 8 12 24 19
10 18 23 17 20 10 21 16 1 19 24
3 21 24 9
4 8 21 2 4
6 16 17 21 7 12 30
12 24 15 17 9 2 9 18 21 11 8 6 11
12 22 25 8 7 4 7 6 16 9 2 1 24
4 21 23 30 20
6 15 4 6 15 27 7
68 18
11 1 15 19 18 28 14 17 5 9 24 10
14 24 11 16 9 29 28 23 20 17 13 4 12 10 18
8 8 20 9 24 12 24 13 30
11 18 16 17 18 1 27 3 16 29 18 17
7 15 9 8 23 13 3 26
1 21
3 30 2 4
2 13 19
6 4 1 7 12 18 24
6 6 18 20 4 5 28
1 11
14 22 25 19 25 20 3 7 6 24 30 9 9 4 19
6 8 20 26 19 7 19
12 12 6 25 15 2 22 29 12 20 21 28 8
7 17 2 13 23 25 12 23
8 15 12 23 22 23 19 2 29
11 16 2 5 2 16 28 24 7 2 13 27
5 12 3 8 13 15
70 7
10 14 2 10 17 17 24 9 5 17 29
7 24 30 29 26 8 19 11
8 20 23 2 10 26 26 17 8
2 8 6
9 14 30 28 30 16 13 1 12 30
5 6 23 21 5 10
8 23 20 3 4 4 4 13 29
59 20
4 30 29 4 18
12 25 15 4 10 20 26 13 19 16 18 3 6
8 13 25 6 24 27 1 28 30
2 26 20
9 24 11 3 19 28 7 13 5 2
6 24 27 26 4 12 13
2 18 27
14 12 24 4 1 25 23 22 30 10 12 4 25 14 28
9 12 26 17 8 27 22 1 23 9
8 27 22 1 6 10 3 9 3
14 9 19 12 1 18 21 4 13 16 17 3 20 28 28
9 5 17 20 27 1 29 22 27 12
1 2
5 9 11 15 28 11
6 9 11 13 29 14 18
12 1 20 26 20 9 24 17 25 5 5 18 3
5 14 14 3 16 26
7 18 3 8 28 6 8 1
12 6 14 27 12 6 8 7 26 17 22 4 3
6 8 20 29 26 26 4
86 13
2 30 20
8 29 9 21 6 9 30 4 15
1 15
4 26 13 7 4
10 10 7 29 10 18 19 5 13 15 25
5 6 16 26 21 7
3 11 4 27
4 7 11 28 13
5 16 25 29 19 20
11 17 19 10 5 29 6 9 13 30 16 10
14 3 30 14 21 10 17 18 12 24 28 1 6 12 16
9 11 27 12 3 13 22 12 9 21
7 18 25 9 25 5 16 19
99 19
4 6 15 27 9
8 16 9 28 28 17 20 30 13
9 24 25 15 5 26 5 13 13 21
11 29 25 6 17 21 26 26 18 11 14 26
2 22 26
9 11 4 10 2 16 3 25 3 17
11 28 13 3 2 3 16 22 20 13 8 2
11 26 19 11 1 6 12 22 1 3 24 5
6 25 12 14 19 6 22
1 3
12 12 26 21 27 17 10 1 25 3 9 12 21
1 4
8 23 26 18 25 19 14 28 6
8 4 24 1 17 3 26 13 6
9 3 24 30 4 24 16 28 25 27
10 13 30 5 27 17 14 13 28 28 11
5 15 6 18 7 22
2 24 26
3 15 28 2
62 13
10 21 20 20 9 29 24 30 25 13 17
3 25 6 20
5 30 4 2 17 2
14 29 26 11 7 2 1 9 7 15 4 27 27 15 27
1 8
6 12 20 4 4 7 9
9 25 8 26 19 17 19 4 7 14
6 13 15 6 13 13 21
8 9 17 22 5 3 22 23 6
1 26
3 9 5 23
10 4 18 14 12 29 17 18 12 22 23
1 28
57 1
2 13 10
81 14
12 12 17 20 10 20 16 10 20 12 3 15 7
2 28 19
2 6 28
14 28 12 10 17 17 10 26 21 11 5 16 9 16 3
6 25 14 13 5 3 16
13 10 23 18 29 3 28 5 22 19 24 4 20 10
6 29 27 2 2 1 17
4 16 19 7 3
12 12 7 27 27 13 28 11 22 26 5 19 22
11 29 15 30 10 16 11 9 12 12 2 4
10 11 12 9 9 14 2 20 20 28 9
10 17 11 15 13 16 26 4 4 16 19
1 26
6 5 26 8 8 27 11
94 17
10 5 15 27 7 27 16 26 27 1 13
5 16 17 14 3 20
3 18 30 3
1 26
9 8 25 6 4 5 3 2 19 29
5 7 27 27 23 23
3 15 27 13
10 13 19 2 24 27 20 24 30 24 19
10 2 5 26 27 9 28 29 19 18 29
4 15 25 10 7
11 24 25 21 16 7 9 9 30 6 20 15
9 14 25 16 7 29 11 3 30 30
1 10
6 21 27 2 7 6 30
11 22 24 28 29 30 6 29 21 3 19 6
5 2 22 7 30 21
1 2
77 19
3 29 26 7
8 27 14 23 26 28 6 11 25
2 10 23
11 22 17 13 19 8 6 11 6 5 1 14
10 12 22 14 2 9 21 19 27 4 12
2 23 9
14 18 5 25 2 21 17 18 25 27 17 30 7 14 26
9 20 15 2 11 29 4 12 19 22
12 22 25 14 6 26 7 23 22 24 16 12 10
12 6 28 4 27 5 18 22 25 7 28 27 9
8 22 12 14 5 12 27 22 25
5 17 1 17 8 24
5 19 25 20 16 23
3 12 19 2
13 13 30 22 1 1 18 22 12 23 27 23 12 18
11 6 26 10 14 3 25 8 13 12 19 28
14 4 9 14 27 3 18 27 24 19 27 3 2 30 26
4 15 29 7 1
6 24 2 17 26 27 25
83 10
1 27
7 16 27 8 5 29 17 1
13 5 19 17 29 18 4 18 2 2 17 25 5 10
2 14 28
12 8 27 5 12 24 29 20 12 28 24 11 15
8 17 11 4 25 9 13 29 19
1 22
13 23 19 6 10 2 3 16 1 30 12 12 15 11
5 26 30 8 28 14
11 14 17 18 9 17 23 29 5 29 20 1
99 8
7 22 9 9 29 1 30 11
7 14 13 27 2 12 5 29
8 19 13 26 7 13 12 29 11
11 19 30 1 1 30 7 14 30 8 5 1
11 7 5 13 11 1 14 22 27 12 1 16
14 26 22 28 30 12 30 30 30 30 30 22 21 7 5
14 6 1 20 4 7 24 8 17 17 21 1 13 25 23
2 10 11
79 10
14 22 29 9 13 20 30 4 10 28 1 22 3 1 11
13 30 27 14 8 13 27 8 25 13 23 15 22 3
7 13 4 9 4 4 13 15
11 9 17 1 1 8 3 23 10 1 22 6
3 30 18 25
9 5 7 21 19 20 15 5 3 18
13 28 21 18 12 24 26 20 16 26 27 18 18 7
5 2 4 9 23 22
7 21 18 9 4 6 20 18
14 14 6 7 11 18 24 15 3 11 4 18 6 23 28
40 11
2 16 24
6 30 7 10 21 24 10
10 21 29 3 23 13 30 29 15 10 22
5 12 24 25 30 29
1 19
13 7 18 13 22 25 12 29 26 24 14 27 9 5
5 4 19 21 3 17
14 4 30 26 16 23 20 7 13 6 17 9 4 4 21
6 20 25 16 15 18 29
13 19 25 19 22 13 10 16 21 7 20 12 2 27
10 22 25 30 19 11 9 23 14 21 10
52 7
5 9 17 15 3 5
10 13 26 14 22 3 27 20 22 30 22
6 26 13 4 17 1 14
7 15 19 8 24 13 6 10
3 22 25 15
14 26 28 13 1 11 16 19 1 29 19 22 9 14 26
8 30 18 18 17 3 28 16 18
96 11
12 23 5 14 8 30 9 27 5 1 29 20 20
7 10 8 12 10 13 7 14
4 25 23 12 19
12 27 28 30 10 25 23 14 30 22 5 8 10
7 1 8 28 20 7 30 19
14 9 1 17 22 26 3 14 8 13 4 4 10 3 5
1 17
6 26 30 23 26 9 1
2 17 21
7 15 20 25 24 28 17 10
12 12 4 25 19 16 20 15 26 15 19 22 1
77 17
7 21 12 1 13 7 17 3
11 23 14 29 17 12 7 26 23 19 22 9
1 7
9 13 24 13 2 7 5 30 24 27
8 27 20 24 26 6 27 30 21
4 21 7 13 27
4 27 15 15 6
4 14 26 18 7
12 19 14 4 10 29 1 21 25 20 15 20 17
5 12 7 4 2 5
9 28 29 5 5 13 2 18 18 27
1 17
2 16 30
12 25 20 30 8 14 11 22 26 19 16 29 18
13 30 22 27 19 20 2 23 25 3 10 12 22 7
2 17 22
3 16 8 30
42 5
10 9 28 1 28 13 21 15 24 20 28
12 8 18 13 1 4 8 2 7 29 8 27 15
1 7
1 21
13 29 27 12 30 25 4 19 29 24 3 22 5 22
48 4
7 24 4 4 1 28 3 21
5 29 28 18 5 20
2 3 10
12 14 9 20 9 27 19 2 21 2 28 13 13
84 3
7 27 9 21 24 11 4 21
5 1 30 27 20 29
6 29 24 5 30 6 14
88 16
7 9 17 5 21 22 28 25
10 24 25 24 18 5 27 8 27 27 29
12 8 19 15 7 4 19 28 9 24 23 16 23
9 25 19 13 8 16 8 19 9 2
12 26 29 1 25 25 19 15 10 27 3 24 25
1 13
4 7 28 6 23
8 7 17 30 11 24 15 18 4
14 12 7 11 2 7 28 26 18 12 6 6 15 21 30
13 25 13 27 23 18 11 4 16 27 4 27 12 10
8 15 25 17 22 6 18 20 3
6 29 6 3 4 20 23
3 2 18 7
3 2 25 1
7 10 20 30 28 1 2 4
12 26 20 21 23 30 2 17 27 1 23 29 4
79 3
11 6 30 5 18 1 21 10 28 23 29 19
10 22 20 23 21 8 5 11 30 26 12
9 22 12 1 12 8 4 24 6 1
90 12
10 16 15 20 5 7 10 24 26 1 5
5 13 12 7 23 4
2 27 20
5 30 12 26 7 15
7 4 8 26 28 25 12 12
12 8 18 16 1 5 16 28 7 21 9 13 13
4 6 9 23 20
3 4 7 7
7 18 3 18 13 30 4 16
5 10 24 28 25 16
10 2 13 1 22 14 13 27 17 19 27
12 30 27 27 7 4 29 24 28 16 28 27 12
58 17
3 29 26 7
6 11 30 19 11 14 24
4 10 10 4 6
1 3
7 22 1 20 20 24 17 6
14 13 17 27 11 29 25 28 5 30 8 26 18 19 9
13 4 18 13 29 16 7 23 2 28 24 21 18 9
7 23 22 11 1 18 13 29
6 10 25 4 17 20 13
3 29 24 22
10 28 12 15 4 5 16 2 20 28 11
12 26 3 12 28 25 21 10 23 25 19 9 20
12 28 2 16 18 17 30 18 14 11 24 10 7
2 3 26
9 13 16 17 7 19 14 1 9 16
7 3 26 23 14 23 12 16
7 22 2 30 1 8 2 24
70 9
7 11 26 8 23 3 24 29
13 29 29 30 14 13 24 10 27 8 24 8 23 24
1 16
3 21 15 17
7 23 17 1 3 12 8 25
9 23 15 5 22 13 4 5 17 20
14 13 27 8 13 11 23 3 18 8 24 3 16 8 25
12 30 28 13 30 14 19 22 29 24 13 3 19
4 20 8 23 24
40 12
12 6 15 9 24 14 24 26 29 1 12 22 22
11 26 21 15 15 13 13 30 25 7 18 4
8 18 19 20 13 10 17 11 16
11 4 29 18 21 19 10 2 10 2 25 6
14 9 12 26 13 11 13 19 20 8 7 7 26 26 12
14 12 22 21 29 17 12 8 7 30 10 30 2 11 24
1 16
14 10 12 28 20 16 17 9 23 15 8 19 10 19 16
11 10 6 3 18 9 11 24 1 20 15 2
6 1 22 29 24 1 10
6 20 25 22 20 10 6
13 28 7 7 13 11 8 10 14 25 11 16 18 11
74 15
10 10 25 25 8 10 25 10 23 6 26
6 17 5 11 14 2 17
12 6 20 20 16 3 14 26 18 24 28 6 18
9 15 4 17 22 13 3 23 28 8
9 25 24 23 6 29 17 14 11 22
5 30 7 27 5 24
12 28 21 11 15 13 25 10 29 9 15 1 1
4 30 19 6 16
4 3 6 20 9
2 11 11
11 10 8 4 25 13 1 16 16 8 20 10
1 19
2 1 11
7 4 11 21 2 26 16 4
1 5
78 19
14 7 7 17 14 10 11 18 3 26 3 10 8 5 18
14 6 19 6 8 22 8 21 15 3 6 11 26 11 14
12 18 20 2 4 25 4 14 5 28 2 29 7
9 25 24 26 1 4 1 8 18 1
14 2 25 4 12 20 14 26 7 1 7 1 26 2 4
1 6
7 10 26 29 10 21 14 5
9 18 28 28 5 28 26 6 22 21
2 4 26
11 10 18 11 10 13 4 5 13 1 27 23
11 17 2 17 1 29 7 10 26 26 14 23
7 11 6 3 20 1 28 16
11 7 19 12 19 22 9 2 15 5 16 11
12 9 27 14 7 25 23 2 20 28 16 3 8
2 5 20
14 25 5 16 1 15 28 20 7 6 13 21 2 28 23
5 6 19 20 5 13
10 28 2 1 6 26 1 19 1 20 24
3 16 2 17
54 19
2 29 26
2 11 19
13 3 4 5 13 15 1 25 19 28 18 19 25 14
9 5 6 22 29 22 8 30 8 30
6 6 20 7 15 30 25
14 2 20 18 14 5 18 30 15 7 9 3 2 14 6
4 11 27 26 2
10 17 10 25 6 15 14 12 29 13 29
14 6 10 21 11 6 30 2 20 7 11 15 30 16 20
7 27 16 23 28 12 9 29
14 7 13 12 10 4 16 30 7 14 10 28 24 15 27
4 5 25 28 11
2 5 30
7 1 15 5 21 18 5 11
12 3 24 19 5 19 4 4 25 9 13 22 25
10 11 12 16 5 1 26 21 5 17 3
14 23 29 10 11 25 20 18 28 5 6 2 23 1 27
8 10 2 23 4 21 3 15 28
14 7 23 20 11 1 14 30 23 12 9 3 28 29 20
88 5
7 18 10 18 7 19 19 8
5 22 20 6 28 17
7 4 1 8 6 1 21 6
1 24
14 26 21 26 7 8 21 24 17 30 3 23 10 21 22
62 4
13 9 1 27 27 26 19 4 23 19 16 20 12 9
10 29 30 13 27 29 3 20 15 24 22
8 25 12 21 29 15 24 29 7
8 26 24 9 29 9 19 15 28
61 15
3 20 14 28
3 4 22 28
1 8
10 10 2 23 22 22 30 15 21 28 27
6 14 27 6 22 15 20
7 7 5 26 26 10 15 4
8 6 2 16 13 13 25 7 5
4 28 26 23 18
6 11 25 7 7 1 28
13 12 8 11 8 25 29 10 9 2 15 15 3 30
9 15 16 17 12 23 7 7 7 24
4 17 11 28 15
1 17
2 22 17
12 30 11 28 9 20 30 15 26 2 14 6 9
54 3
2 21 29
14 20 14 10 6 24 7 13 26 16 2 10 24 1 1
10 29 1 15 20 15 2 13 28 8 21
81 20
10 8 20 20 19 25 29 25 19 27 7
14 12 8 15 5 30 15 1 20 7 7 9 21 9 22
14 16 4 27 7 6 27 26 17 15 20 15 9 30 11
11 28 15 6 12 11 6 19 3 25 25 10
6 8 18 17 17 25 12
14 1 9 10 26 25 24 7 1 25 29 4 1 26 18
9 29 20 4 17 15 20 4 24 15
3 3 23 19
9 5 2 19 13 11 6 30 27 12
6 21 10 3 13 27 12
3 26 2 14
10 8 4 29 23 10 1 25 3 20 13
5 13 23 3 16 28
12 12 1 24 2 3 26 14 29 8 25 16 1
2 20 8
9 18 30 5 19 16 29 30 20 19
8 12 13 27 1 15 8 1 8
5 25 25 7 16 2
7 1 24 14 20 1 9 8
10 5 18 29 25 17 18 13 28 21 25
40 13
11 24 14 30 25 30 25 23 15 18 15 16
14 28 5 4 28 4 26 24 21 24 18 7 4 22 27
6 9 21 7 9 6 12
7 30 3 25 14 18 4 20
11 15 10 21 19 7 25 6 1 7 29 10
2 24 2
2 10 10
2 16 10
7 19 10 11 13 26 24 30
4 14 16 6 15
5 24 21 23 29 13
9 20 23 18 13 24 11 22 25 24
13 4 5 9 6 7 22 1 22 13 22 27 29 28
73 7
13 24 11 4 6 1 15 20 11 19 13 21 11 29
11 1 3 11 2 30 9 15 30 30 27 14
1 17
1 7
11 7 30 15 10 28 8 24 17 18 13 22
14 15 20 6 7 14 16 30 13 24 14 5 16 3 18
2 19 12
80 6
6 10 22 28 29 21 13
7 30 17 6 30 23 26 5
12 1 13 29 14 28 5 18 13 7 5 24 17
8 4 2 25 5 24 22 3 6
7 9 6 13 15 5 28 2
9 19 2 13 9 15 11 13 24 23
78 20
9 6 27 4 7 22 8 22 5 2
10 1 10 25 6 16 29 3 17 29 21
13 4 29 17 6 12 11 28 22 22 28 27 19 1
10 10 30 9 14 23 28 15 25 22 12
2 12 6
3 3 26 21
12 16 7 11 19 9 30 11 1 20 29 19 12
12 20 4 25 3 26 22 9 20 6 12 22 17
7 10 19 4 22 16 12 20
10 30 29 18 2 21 7 23 1 18 8
4 21 24 14 9
8 15 28 13 26 11 29 13 12
8 8 3 25 19 23 21 11 13
8 12 3 6 26 25 23 26 6
2 19 11
14 26 25 3 8 13 14 7 17 25 16 24 28 10 13
6 22 23 24 21 26 18
4 22 12 11 17
3 8 27 11
13 23 6 15 22 10 28 28 26 15 5 19 4 7
97 15
8 7 30 11 3 17 28 16 20
10 24 20 7 20 30 19 4 27 4 26
8 23 15 23 29 12 12 24 18
10 30 29 3 29 9 27 15 29 12 26
6 27 15 27 16 15 7
4 3 10 7 9
5 21 1 23 24 4
11 11 22 16 2 24 7 10 20 13 30 23
7 20 19 15 8 26 29 14
7 2 16 5 2 2 18 24
8 11 28 2 14 19 10 15 4
8 16 15 20 16 7 20 27 17
13 26 4 25 1 3 26 8 7 19 9 16 13 25
7 2 18 2 20 19 16 15
10 1 29 8 8 27 27 26 13 23 13
58 18
4 10 13 13 9
2 22 16
2 8 4
13 17 5 18 28 20 2 15 13 22 14 20 18 11
4 22 25 20 29
14 25 1 17 8 1 10 29 16 7 28 20 5 14 24
10 3 14 15 18 18 6 1 7 23 3
12 14 28 26 4 9 20 4 25 27 4 26 17
12 2 6 1 6 20 16 19 14 21 3 1 8
6 24 7 14 26 12 19
9 7 23 16 19 18 2 7 14 28
14 25 21 29 25 27 10 10 7 24 1 2 16 30 1
5 6 7 27 18 25
3 16 9 26
8 27 20 11 10 17 26 4 7
8 20 25 26 21 2 11 13 3
4 13 25 27 10
11 23 27 18 25 5 26 13 30 14 2 10
89 20
1 18
14 13 29 3 30 20 26 2 24 20 19 6 14 15 16
5 29 4 16 16 8
9 20 8 25 21 9 9 22 28 27
9 10 25 23 1 6 10 2 30 29
2 27 5
9 4 3 25 8 18 10 7 20 29
4 14 11 7 23
2 4 11
11 5 5 7 5 3 9 6 2 7 25 28
3 21 2 27
12 1 14 17 7 4 8 5 9 10 11 23 4
4 3 18 18 30
12 22 2 2 19 25 1 5 22 25 18 15 22
3 15 27 3
8 22 10 10 23 20 20 15 23
9 10 2 12 1 25 25 24 19 5
8 11 9 1 27 26 16 10 12
14 7 15 27 20 24 28 12 5 17 19 19 11 28 20
10 20 7 30 13 17 4 30 19 5 22
57 2
5 25 4 13 23 18
5 12 11 29 16 8
41 14
10 18 23 16 1 12 14 23 16 30 26
11 18 30 28 24 22 27 18 25 9 10 4
9 14 14 29 21 21 14 24 9 23
6 24 23 11 29 7 26
4 3 2 7 24
5 30 7 9 10 23
7 11 26 10 16 2 8 6
10 21 29 22 5 28 7 28 8 5 26
9 24 28 18 30 14 1 22 12 9
3 27 18 3
14 27 11 15 26 16 29 9 7 20 13 4 27 2 4
13 28 20 17 17 8 8 30 8 21 4 8 13 30
7 8 13 13 18 28 9 3
6 9 9 7 21 5 3
75 20
14 20 19 4 6 26 11 28 25 24 23 3 6 22 19
1 26
6 22 23 2 17 10 10
3 17 22 21
1 14
4 28 25 1 1
13 26 4 20 13 27 4 7 24 17 25 29 12 18
10 27 11 6 6 12 23 14 4 5 25
13 17 14 4 17 7 26 4 10 7 16 28 10 14
7 26 9 11 29 18 23 25
9 20 1 2 4 14 27 9 30 6
3 14 1 3
10 26 28 21 24 6 10 3 19 22 20
1 3
12 15 17 13 5 6 5 28 10 18 25 18 10
2 26 23
12 28 4 9 25 16 2 30 25 26 19 8 15
7 2 24 13 18 6 9 24
6 7 25 20 23 4 21
6 29 5 27 18 30 5
78 5
8 4 1 23 14 8 7 12 2
10 17 19 28 25 4 29 23 28 10 15
11 30 20 21 4 8 8 3 12 12 9 1
1 9
7 20 9 29 23 10 21 9
