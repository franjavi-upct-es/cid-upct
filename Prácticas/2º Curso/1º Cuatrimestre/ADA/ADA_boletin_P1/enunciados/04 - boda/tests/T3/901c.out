no solution
56
89
79
94
no solution
70
no solution
no solution
no solution
no solution
13
no solution
94
no solution
no solution
99
79
no solution
52
96
no solution
42
48
84
88
79
90
no solution
70
no solution
74
78
no solution
88
62
no solution
54
81
no solution
73
80
no solution
97
no solution
89
54
no solution
no solution
78
